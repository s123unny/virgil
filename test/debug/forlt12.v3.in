b ../core/forlt12.v3 7
run
q
