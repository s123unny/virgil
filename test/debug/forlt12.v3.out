breakpoint|../core/forlt12.v3 7|0
end
result|!NullCheckException
	in main() [../core/forlt12.v3 @ 4:23]

